.github/CODEOWNERS

```diff
@@ -0,0 +1,20 @@
+# anything with no explicit code owners will be tagged to @core-devs
+*                       @napari/core-devs
+
+# submodules
+napari/_vispy/          @brisvag @melonora
+napari/_qt/             @Czaki @lucyleeow @DragaDoncila
+napari/_app_model/      @lucyleeow @DragaDoncila
+napari/benchmarks/      @Czaki @jni
+napari/plugins/         @Czaki @DragaDoncila @lucyleeow
+napari/qt/              @Czaki @jni
+napari/settings/        @Czaki @jni
+
+# specific layers
+napari/layers/image/    @Czaki @brisvag @andy-sweet
+napari/layers/labels/   @jni @Czaki @brisvag
+napari/layers/points/   @brisvag @kevinyamauchi @andy-sweet @DragaDoncila
+napari/layers/shapes/   @kevinyamauchi @DragaDoncila @melonora
+napari/layers/surface/  @brisvag @kevinyamauchi @Czaki
+napari/layers/tracks/   @Czaki @andy-sweet
```

Comment:
I know nothing about tracks. I only do general refactor there. So it looks like this module is really abadonded...

Reply:
Feel free to add me to abandoned modules. They can all be understood with a little elbow grease. 😊